src/reports/reports.service.ts

```
@@ -20,93 +23,56 @@ export class ReportsService {
+        })),
+      },
     };
 
     try {
       const crew = await this.crewsService.findOne(req, crew_id);
       if (!crew) throw new NotFoundException('Crew Not Found');
-
-      newReportData.served_by = crew.data.name;
-      newReportData.collected_by = crew.data.name;
     } catch (error) {
       throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)